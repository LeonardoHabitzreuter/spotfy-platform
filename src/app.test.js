import React from 'react'
import { shallow } from 'enzyme'
import App from './app'

describe('alert component', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot()
  })
})
