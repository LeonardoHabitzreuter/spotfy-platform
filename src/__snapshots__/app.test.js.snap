// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`alert component should render correctly 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      exact={true}
      path="/"
      render={[Function]}
    />
    <Route
      path="/artists"
      render={[Function]}
    />
    <Route
      path="/albums"
      render={[Function]}
    />
    <Route
      path="/tracks"
      render={[Function]}
    />
  </Switch>
</BrowserRouter>
`;
